pkg/controllers/status/controller.go

```diff
@@ -488,9 +498,8 @@ func (s *KubeFedStatusController) clusterStatuses(
 			s.typeConfig.GetTargetType(),
 		)
 		if err != nil {
-			wrappedErr := errors.Wrapf(err, "Failed to get %s %q from cluster %q", targetKind, key, clusterName)
-			runtime.HandleError(wrappedErr)
-			return nil, wrappedErr
+			keyedLogger.WithValues("target-kind", targetKind, "cluster-name", clusterName).Error(err, "Failed to get object from cluster")
```

Comment:
`target-kind` is unnecessary (across the controller) since the ftc name already conveys the target group-kind.

Reply:
done, pls take review.